framework/auth/cas.py

```diff
@@ -55,20 +55,15 @@ class CasClient(object):
     def __init__(self, base_url):
         self.BASE_URL = base_url
 
-    def get_login_url(self, service_url, auto=False, username=None, password=None, verification_key=None, otp=None):
+    def get_login_url(self, service_url, username=None, verification_key=None):
         url = furl.furl(self.BASE_URL)
         url.path.segments.append('login')
         url.args['service'] = service_url
-        if auto:
+        if username and verification_key:
+            # TODO: remove auto=True from url when new CAS (PR#18) is deployed
```

Comment:
Go ahead and remove the `auto` option, we will be releasing both at the same time